contracts/strategies/allocation/donation-voting-passport-offchain-calc/README.md

```diff
@@ -0,0 +1,57 @@
+Spec: Donation Voting with Passport and Offchain Calculation
+---------------------------------
+
+## Overview 
+There is a category of mechanisms (including Quadratic Funding or QF) allocate a funding pool based upon individual donations to eligible recipients. Essentially, the amount that an individual donates to a project is considered their "vote", which is weighted according to the type of voting calculation formula being used. 
+
+At time of writing (June 2023), most prominent QF formulas are too computationally expensive to be calculated on-chain at a reasonable cost. As a step towards an eventual fully on-chain QF solution, this strategy functions as a hybrid solution: 
+* votes are recorded **on-chain**
+* the allocation is calculated **off-chain**
+* the final distribution is recorded **on-chain**
+
+This strategy also uses Passport as a voter eligibility signal as outlined below. 
+
+## Spec
+### Recipient logic
+In this strategy, recipients must apply and be approved by pool admins. 
```

Comment:
are there any start/end dates for register and allocation?
(I guess yes: but they should be part of the spec)